roles/test-sync-app-apb/tasks/unbind.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+- name: "Test|Check that sync binding secret exists"
```

Comment:
should be `... binding secret does not exist`?

Reply:
+1

Reply:
@psturc Updated